frontend/src/views/LoginView.vue

```diff
@@ -1,13 +1,66 @@
 <template>
 	<div class="page_box_wrapper">
 		<div class="page_box">
-			<h1>This is a login page</h1>
+          <!-- Content should be here -->
+		  <h1> HI WANNA PLAY PONG? </h1>
+		  
+		  <div class="c_block c_form_group">
+			  <form 
+			  method="post"
+					action=""
+					class="c_block c_form_group"
+					@submit.prevent="userStore.login(loginForm)"
+					>
+					
+					<InputField label="Username" placeholder="username" v-model="loginForm.name"/>
+					<InputField label="Password" placeholder="password" v-model="loginForm.password"/>
+					
+					<div class="c_block c_split">
+						<p>
+							<input v-on:submit="userStore.login(loginForm)" class="link_button" type="submit" value="Login" />
+							/
+							<a href="/register">Register</a>
+						</p>
+						<p>
+							<a :href="redirectUrl">Sign in with 42</a>
+						</p>
+					</div>
```

Comment:
looks nice, would be cool to handle errors when the username and password are wrong or empty for example

Reply:
Yes indeed! At the moment it does throw an alert for non matching usern or passw, or empty. I opened a new ticket for a more verbose error handling

Reply:
wonderfull!